Patches/cppzmq/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 2.8)
```

Comment:
We probably don't need this patch anymore. The latest version provides a CMakeLists.txt file. Let's start there. If it doesn't work, we can patch it and try to push fixes upstream.

Reply:
I removed the patch and ran into this build error:

CMake Error at /usr/share/cmake-3.10/Modules/WriteBasicConfigVersionFile.cmake:40 (message):
  No VERSION specified for WRITE_BASIC_CONFIG_VERSION_FILE()
Call Stack (most recent call first):
  /usr/share/cmake-3.10/Modules/CMakePackageConfigHelpers.cmake:207 (write_basic_config_version_file)
  CMakeLists.txt:53 (write_basic_package_version_file)

-- I assume this error is the reason the patch is there; so I kept the patch in the next commit.